src/app/landing-page/info-box/info-box.component.scss

```diff
@@ -1,17 +1,15 @@
 @import "../../../colors.scss";
-
+@import "../../../styles.scss";
 
 .info-container {
-  padding: 24px;
-  width: 250px;
-  height: 380px;
-}
-
-.info-content {
-  width: 100%;
-  height: 100%;
-  >h2 {
-    color: $primary;
+  padding: 20px;
+  .info-content {
+    width: 100%;
+    height: 100%;
+    h1 {
```

Comment:
Perhaps instead of using a tag selector like `h1`, it would be wise to add a class to the title element like `<h1 class="info-content-title">` and use the `.info-content-title` selector. It would prevent unnecessary changes when you change the tag from `<h2>` to `<h1>`. If it used the class selector instead of the tag selector, you would only need to change the HTML; the CSS will remain the same as you would use the same class name.